sanic/app.py

```diff
@@ -27,7 +27,7 @@
 class Sanic:
 
     def __init__(self, name=None, router=None, error_handler=None,
-                 load_env=True, request_class=None,
+                 load_env=True, request_class=None, is_request_stream=False,
```

Comment:
I think it is better to add `self.is_request_stream = True` in `app.route()` for `blueprint.stream()`. 

```
if stream:
    self.is_request_stream = True
```